src/services/expenses.service.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable function-paren-newline */
```

Comment:
Can we fix this eslint issue? 

(if we can't, let's disable it for the whole app using the config)